### DISCUSSION TITLE
Still running 1.19.3 Stopped gathering data 4 days ago

### DISCUSSION THREAD

**Discussion by houseofdreams** [teslamate-org/teslamate#2323] [Q&A]
Hi als the title says, I'm still running an older version (1.19.3) and all of a sudden, teslamate stopped gathering new data (no new drives, ....)

I Installed it, not using docker , but manually (debian tutorial). When I now try to log on to the teslamate dashboard (port 4000) I enter the username/password combo and I get the error "Error: unknown"

I have no idea what is wrong, older data is still available in grafana. 

Is there an upgrade tutorial for people not using docker? (I use debian because I have a custom made front end website, with the look&feel of teslascope)

Any help appreciated!

**Comment by eliluong** [comment]
It could be related to https://github.com/adriankumpf/teslamate/issues/1945 and you would need to update to whichever release allowed you to input the tokens.
